mne/epochs.py

```diff
@@ -1233,6 +1242,15 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
+                if any([k_e not in epochs.event_id for k_e in key]):
+                    key = [k for k in epochs.event_id.keys()
+                           if all([set(k_i.split('/')).issubset(k.split("/"))
+                                   for k_i in key])]
```

Comment:
nope, and that's intended I think -- I mean that `auditory/left` is and probably should be treated the same as `left/auditory`